src/design-guidelines/shadows/shadows-and-depth.md

```diff
@@ -55,25 +59,29 @@ It's crucial that an interaction (like hovering or pressing) that changes a visu
         transform: translate3d(0, 0.08rem, 0);
     }
 }
-````
+```
+
 Also note how we use `transform` to make our buttons feel like they are being pushed down, when pressed by the user.
 :::
 
 ### 2. Shadow variables for surfaces
+
 In a digital user interface, there are many floating UI elements like cards, modals, menus, or popovers that can benefit from a shadow effect. Here is a list of the various CSS variables for shadows that Lime Elements offers:
 
 <limel-example-surface-shadows></limel-example-surface-shadows>
 
 ### 3. Shadow variables for highlighting different states
+
 <limel-example-surface-shadows-states></limel-example-surface-shadows-states>
```

Comment:
<img width="695" alt="image" src="https://user-images.githubusercontent.com/154725/211840795-483a54eb-bdc3-414c-8459-0a4eb70b57ed.png">

I never noticed it in light mode, but I just have to point out how god damn good it looks that the drop shadow is on top of the focus ring here! 😍 Chef's kiss! 👌

Reply:
I'm glad you noticed it. Yes, the effect becomes really sexy. Specially the red one with the errors ;)